doc/misc/npm-config.md

```diff
@@ -879,6 +879,14 @@ If true, output the npm version and exit successfully.
 
 Only relevant when specified explicitly on the command line.
 
+### version-prefix
```

Comment:
This is ambiguous, and should be either `tag-prefix` or (preferably) `tag-version-prefix`.